plugins/askrene/layer.c

```diff
@@ -319,73 +347,81 @@ void layer_add_localmods(const struct layer *layer,
 		if (!node)
 			continue;
 		for (size_t n = 0; n < node->num_chans; n++) {
-			struct short_channel_id scid;
+			struct short_channel_id_dir scidd;
 			struct gossmap_chan *c;
-			int dir;
-			c = gossmap_nth_chan(gossmap, node, n, &dir);
-			scid = gossmap_chan_scid(gossmap, c);
+			bool enabled = false;
+			struct amount_msat zero = AMOUNT_MSAT(0);
+			c = gossmap_nth_chan(gossmap, node, n, &scidd.dir);
+			scidd.scid = gossmap_chan_scid(gossmap, c);
 
 			/* Disabled zero-capacity on incoming */
 			gossmap_local_updatechan(localmods,
-						 scid,
-						 AMOUNT_MSAT(0),
-						 AMOUNT_MSAT(0),
-						 0,
-						 0,
-						 0,
-						 false,
-						 !dir);
+						 &scidd,
+						 &enabled,
+						 &zero, &zero,
+						 NULL, NULL, NULL);
```

Comment:
Now that we have the ability to select which fields we want to change, why are we passing `zero` values to `htlc_min` and `htlc_max`?